api/src/opentrons/protocol_engine/types.py

```diff
@@ -671,16 +671,27 @@ class PotentialCutoutFixture:
     cutout_fixture_id: str
 
 
+class AreaType(str, Enum):
```

Comment:
Can we avoid subclassing this from `str`? [Rationale here.](https://opentrons.atlassian.net/wiki/spaces/RPDO/pages/3479896162/Don+t+subclass+Python+enums+from+str+proposed)

```suggestion
class AreaType(Enum):
```